app/view_models/nsm/v1/work_item_summary.rb

```diff
@@ -1,13 +1,50 @@
 module Nsm
   module V1
-    module WorkItemSummary
+    class WorkItemSummary < BaseViewModel
+      attribute :submission
+
+      def header
+        [
+          t('items', numeric: false),
+          t('requested_time'),
+          t('requested_cost'),
+          t('allowed_time'),
+          t('allowed_cost')
+        ]
+      end
+
+      def table_fields
+        work_item_data.map do |data|
+          format_row(data)
+        end
+      end
+
+      def footer
+        format_row([t('total', numeric: false)] + summed_values(work_items, periods: false))
+      end
+
+      def format_row(data)
+        work_type, requested_cost, requested_time, allowed_cost, allowed_time = *data
+        result = [
+          work_type,
+          { text: format_period(requested_time, style: :line_html), numeric: true },
+          { text: NumberTo.pounds(requested_cost), numeric: true },
+        ]
+        if requested_cost != allowed_cost || submission.part_grant?
+          result << { text: format_period(allowed_time, style: :line_html), numeric: true }
+          result << { text: NumberTo.pounds(allowed_cost), numeric: true }
+        else
+          result << '' << ''
```

Comment:
I don't think this does anything, does it?

Reply:
without the cells aren;t rendered so the table lines don't appear

Reply:
but this line doesn't change the string, does it?

```ruby
a = "foo" 
#>  "foo"
a << "" << ""
#> "foo"
a == "foo"
#> true
```